### PR TITLE
Version 0.1.0 release!

### PR DESCRIPTION
**Pull request by anyaelena**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+## Alfalfa Version 0.1.0
```
Comment (kbenne) [inline CHANGELOG.md:1]: Ok so this is the version of record right? the main place the version is recorded and tracked.

File: alfalfa_web/Dockerfile [hunk 2]
```diff
@@ -17,5 +17,5 @@ COPY lib/dbops.js /root/server/
 
 ARG NODE_ENV
 RUN npm run build
-
+RUN ls -alh build/app/
```
Comment (kbenne) [inline alfalfa_web/Dockerfile:20]: this looks harmless, but im just curious what is the purpose?

File: alfalfa_web/server/index.js [hunk 3]
```diff
@@ -104,15 +104,14 @@ MongoClient.connect(process.env.MONGO_URL).then((mongoClient) => {
...
-        if ( process.env.S3_URL.indexOf("amazonaws") == -1 ) {
-          if (req.hostname.indexOf("alfalfa_web") == -1 ) {
-            // const url =  process.env.S3_URL + '/' + process.env.S3_BUCKET
-            const url = 'http://' + req.hostname + ':9000/' + process.env.S3_BUCKET
-            data.url = url;
-          } else {
-            const url = 'http://minio:9000/alfalfa';
-            data.url = url;
-          }
+        // if you're running locally and using internal Docker networking ( "http://minio:9000")
```
Comment (kbenne) [inline alfalfa_web/server/index.js:107]: yep. I believe we discussed this at length.

File: pyproject.toml [hunk 4]
```diff
@@ -0,0 +1,44 @@
+[tool.poetry]
+name = "alfalfa"
+version = "0.1.0"
```
Comment (kbenne) [inline pyproject.toml:3]: ok. maybe this is the version of record. version of record for computers? changelog being the human facing version.

**Review by kbenne — APPROVED** [review]

**Comment by kbenne** [comment]
One more comment @anyaelena. Nick actually brought this up on scrum. Now that you have this first release at hand, how about we establish a cadence for the future releases? Perhaps add a milestone (or some other tracking) for the future releases. quarterly? twice a year?

File: alfalfa_web/Dockerfile (hunk 2 above)
Comment (anyaelena) [reply alfalfa_web/Dockerfile:20]: Debugging that app.bundle.js weird filesize stuff.  I just decided to leave it in.

File: pyproject.toml (hunk 4 above)
Comment (anyaelena) [reply pyproject.toml:3]: Yes, the docker publishing script currently parses this.

**Pull request merged**